components/List/index.js

```diff
@@ -0,0 +1,38 @@
+import { StyledCard } from "../Card/Card.styled";
+import { StyledSpan } from "../Card/Card.styled";
+import Image from "next/image";
+
+export default function List({ rides }) {
+  return (
+    <ul>
+      {rides.map((ride) => (
+        <StyledCard key={ride.id}>
```

Comment:
How about use <li> here? I notice that, there is no li in ul.

Reply:
you are right, I think in general it is not necessary to use any list here. I turned the `<ul>` into a `<div>`now :)